src/ypd/servlets/project_servlet.py

```diff
@@ -74,10 +74,17 @@ def archive(self, project):
     def submit(self):
         form = SubmissionForm()
 
-        if form.validate_on_submit():
+        if request.method == 'POST':
                 projType = form.projType.data
                 title = form.title.data
                 description = form.description.data
+                electrical = DegreeAttributes.electrical.value in form.degree.data
+                mechanical = DegreeAttributes.mechanical.value in form.degree.data
+                computer = DegreeAttributes.computer.value in form.degree.data
+                computersci = DegreeAttributes.computersci.value in form.degree.data
+                grade = GradeAttributes(form.grade.data)
+                print(f'Grade Value: {grade}')
```

Comment:
We should remove this debugging print, my guy

Reply:
Removed